app/javascript/components/enrollment/steps/Arrival.js

```diff
@@ -91,7 +91,7 @@ class Arrival extends React.Component {
                     id="date_of_departure"
                     date={this.state.current.patient.date_of_departure}
                     minDate={'2020-01-01'}
-                    maxDate={moment().add(30, 'days').format('YYYY-MM-DD')}
+                    maxDate={moment().format('YYYY-MM-DD')}
```

Comment:
I know this isn't being used at the moment, but could you update this too in the meantime? https://github.com/SaraAlert/SaraAlert/blob/master/app/models/concerns/patient_date_validator.rb#L20

Reply:
Sure thing @timwongj! See [0285d5a484b21f702a634a8c791c64105240ddf5](https://github.com/SaraAlert/SaraAlert/pull/1193/commits/0285d5a484b21f702a634a8c791c64105240ddf5)